src/frontends/tensorflow_common/src/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_tensorflow_common")
+get_filename_component(root_dir "${CMAKE_CURRENT_SOURCE_DIR}" DIRECTORY)
+
+file(GLOB_RECURSE LIBRARY_SRC ${root_dir}/src/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${root_dir}/src/*.hpp ${root_dir}/dev_api/*.hpp)
+file(GLOB_RECURSE LIBRARY_PUBLIC_HEADERS ${root_dir}/include/*.hpp)
+
+add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${LIBRARY_HEADERS} ${LIBRARY_PUBLIC_HEADERS})
+target_link_libraries(${TARGET_NAME} PRIVATE openvino::util)
+add_library(openvino::frontend::tensorflow_common ALIAS ${TARGET_NAME})
+
+add_library(${TARGET_NAME}_dev INTERFACE)
+target_link_libraries(${TARGET_NAME}_dev INTERFACE ${TARGET_NAME})
+target_include_directories(${TARGET_NAME}_dev INTERFACE $<BUILD_INTERFACE:${root_dir}/dev_api>)
+add_library(openvino::frontend::tensorflow_common::dev ALIAS ${TARGET_NAME}_dev)
+
+target_include_directories(${TARGET_NAME}
+        PUBLIC
+        ${root_dir}/include
```

Comment:
I suppose we need to use `BUILD_INTERFACE` here to make https://dev.azure.com/openvinoci/dldt/_build/results?buildId=469639&view=logs&jobId=982848f2-7f8d-5e57-730a-8df1e5f991ba&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=b9502550-195c-5289-dc0f-1979026ef2c8 happy

Reply:
Done